clusters/build-clusters/01_cluster/openshift-machine-api/admin_build01-9hdwj-worker-us-east-1a_machine_set.yaml

```diff
@@ -32,9 +32,9 @@ spec:
           apiVersion: awsproviderconfig.openshift.io/v1beta1
           blockDevices:
           - ebs:
-              iops: 32000
-              volumeSize: 640
-              volumeType: io1
+              iops: 0
+              volumeSize: 300
```

Comment:
The performance is going to be really low on these volumes.  Better to use io volumes with smaller iops requests (like 5000) as the performance will be drastically better.